### PR TITLE
Release 0.8

### PR DESCRIPTION
**Pull request by hypothesist**

### CODE REVIEW

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -1,4 +1,135 @@
...
+                                                                                                 
+
+For this release, we have had 45 developers from around the world contributing
+over 148 commits in the last 8 weeks.
+
+Improving the API available for bear writers is one of the areas we've focussed
+on for this release, with several new and exciting features. General performance
+has also been improved heavily with some major changes under the hood. The
+documentation has also been worked on, with an emphasis on user-friendliness
+and design.
```
Comment (sils1297) [inline RELEASE_NOTES.rst]: I'd add that "major internal changes" have been done to prepare bears for being installed individually in the future

Comment (hypothesist) [inline RELEASE_NOTES.rst]: Also introduce the other GSoC products? bear creation tool, quickstart?

Comment (sils1297) [inline RELEASE_NOTES.rst]: those aren't distributed with coala core and will get own release notes

Comment (hypothesist) [inline RELEASE_NOTES.rst]: More release notes?! :P

Comment (sils1297) [inline RELEASE_NOTES.rst]: well they are own packages at the moment so they get own release notes

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -1,4 +1,139 @@
...
+      |                                   |
+      |                  \          \     |
+       \       \         |\          |    |
+        \      /\        | \         |    |
+        |     |  \       |  \        |     \
+        |     |   |     /    |      /\      |
+        |    /    |    |     |     /  |      \
+       /_____}   /_____}    /______}  {_______}
+
+^ Fabian, gimme something better than this :P
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: im sure we won't leave this here

haha, yes, we need something better than this lmao, this bear makes me laugh everytime i see it :p no offense to your drawing talent

File: RELEASE_NOTES.rst [hunk 3]
```diff
@@ -1,4 +1,139 @@
...
+                                                                                                 
+
+For this release, we have had 45 developers from around the world contributing
+over 148 commits in the last 8 weeks.
+
+Improving the API available for bear writers is one of the areas we've focussed
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: focused*

File: RELEASE_NOTES.rst [hunk 4]
```diff
@@ -1,4 +1,139 @@
...
+
+- Bear results may now have a ``confidence`` parameter: this is supposed to
+  quantify the confidence, on a scale of 1 to 100, the bear has when flagging results.
+
+- A ``deprecate_settings`` decorator has been created to deprecate old,
+  unsupported bear parameters. Please see
+  `here <https://github.com/coala-analyzer/coala/blob/fa8fe22562277762fd73ab3761ad1ec33263839a/coalib/bearlib/__init__.py#L15>`_
+  for an example usage.
+
+- A new ``CAN_FIX`` value ``Code Simplification`` has been introduced.
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: rephrase it, sounds like shit

File: RELEASE_NOTES.rst [hunk 5]
```diff
@@ -1,4 +1,139 @@
...
+
+**Documentation**
+
+- A complete overhaul to the README page with a focus on design and
+  readability.
+
+- A new `FAQ page <http://coala.readthedocs.io/en/latest/Users/FAQ.html>`_ has
+  been created.
+
+- Various other documentaion pages have been improved with new resources,
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: documentation

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: No, we're definitely not leaving it there :P

@fneu help please :P

File: RELEASE_NOTES.rst (hunk 4 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: This is a very minor point actually. Debatable whether it should remain in the changelog

Comment (sils1297) [inline RELEASE_NOTES.rst]: I'd rephrase or keep it, when in doubt in release notes more is better IMO.

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (sils1297) [inline RELEASE_NOTES.rst]: he claims to have no time right now, but proposes to try to only make a claw. Maybe try to make a claw with the asciiart style of the previous release arts?

File: RELEASE_NOTES.rst [hunk 6]
```diff
@@ -1,4 +1,140 @@
...
+
+- ``coala-json`` now supports ``--show-bears`` and ``--filter-by-language``
+
+- Added a ``--show-capabilities`` flag that displays the types of issues coala
+  can detect and fix for a particular language.
+
+- Display the line number when a line is missing; this could happen if a bear
+  that had run previously overwrites it.
+
+**For Bear Writers**
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: mention alex's project with writing bears iin any language?

Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: we also got the bear-creation tool of mr-karan, right?

Comment (hypothesist) [inline RELEASE_NOTES.rst]: I asked him, he said he still has that unfixed bug and wants to move it for next release.

Forgot about the bear creation tool, sorry! I'll add that in.

File: RELEASE_NOTES.rst [hunk 7]
```diff
@@ -1,4 +1,140 @@
...
+**Documentation**
+
+- A complete overhaul to the README page with a focus on design and
+  readability.
+
+- A new `FAQ page <http://coala.readthedocs.io/en/latest/Users/FAQ.html>`_ has
+  been created.
+
+- Various other documentation pages have been improved with new resources,
+  better explanations, and some corrections.
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: mention docs are under improvement and we're moving them to a new repo?

File: RELEASE_NOTES.rst [hunk 8]
```diff
@@ -1,4 +1,146 @@
...
+         .ohhs-+mMMMMMMMMMMMMMMMMMMMMMMMMd  
+            .yMMMMMMMMMMMMMMMMMMMMMMMMMMMh  
+            mMMMMMMMMMMMMMMMMMMMMMMMMMMMh`  
+            yMMMMMMMMMMMMMMMNhssssyyyso-    
+             /dMMMMMMMMMNy+.                
+               ./syhys/-                    
+
+
+For this release, we have had 45 developers from around the world contributing
+over 148 commits in the last 8 weeks.
```
Comment (sils1297) [inline RELEASE_NOTES.rst]: did you update the numbers?

Comment (hypothesist) [inline RELEASE_NOTES.rst]: yes, latest from yesterday

Comment (sils1297) [inline RELEASE_NOTES.rst]: I'm sure they changed today again, possibly even another unique contributor

Comment (hypothesist) [inline RELEASE_NOTES.rst]: I'll update after the ack then :+1:

**Comment by Adrianzatreanu** [comment]
rebase :D

**Comment by Adrianzatreanu** [comment]
ack f3566da

**Comment by Adrianzatreanu** [comment]
@rultor merge
